MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
```

Comment:
- [ ] God job with your [MIT.md](http://mit.md/) file :+1:  Please add your name in place of [YOUR NAME], add the current year because it very important o peer attention to this details :smile: